pubsub/google/cloud/pubsub/message.py

```diff
@@ -85,7 +86,10 @@ def from_api_repr(cls, api_repr):
         :rtype: :class:`Message`
         :returns: The message created from the response.
         """
-        data = base64.b64decode(api_repr.get('data', b''))
+        try:
+            data = base64.b64decode(api_repr.get('data', b''))
+        except (binascii.Error, TypeError):
+            data = api_repr.get('data', b'')
```

Comment:
Also, this won't really fix #2513 since the data being sent through the console is not b64encoded. So literally `data` coming in is "testing 1 2 3".

Running the test with the message string `testing 1 2 3` doesn't throw any padding errors.